cxx-squid/src/main/java/org/sonar/cxx/CxxVCppBuildLogParser.java

```diff
@@ -234,10 +228,43 @@ private void ParseCommonCompilerOptions(String line, String fileElement) {
     //__TIME__ The most recent compilation time of the current source file. The time is a string literal of the form hh:mm:ss.
     AddMacro("__TIME__=\"??:??:??\"", fileElement);
     //__TIMESTAMP__ The date and time of the last modification of the current source file, expressed as a string literal in the form Ddd Mmm Date hh:mm:ss yyyy, where Ddd is the abbreviated day of the week and Date is an integer from 1 to 31.
-    AddMacro("__TIMESTAMP__=\"??? ?? ???? ??:??:??\"", fileElement);    
-    //_ATL_VER Defines the ATL version. In Visual Studio 2010, _ATL_VER is defined as 0x0A00.
-    AddMacro("_ATL_VER=1", fileElement);

```

Comment:
Correct. I need to add this again and I will whether I find some more information on the version numbers.
```
_ATL_VER - Defines the ATL version, encoded as an integer literal.
 ```